src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-FURMARK.json

```diff
@@ -0,0 +1,55 @@
+{
+  "Description": "Furmark GPU Performance Workload",
+  "MinimumExecutionInterval": "00:10:00",
+  "Metadata": {
+    "RecommendedMinimumExecutionTime": "00:05:00",
+    "SupportedPlatforms": "win-x64",
+    "SupportedOperatingSystems": "Windows"
+  },
+  "Parameters": {
+    "Time": "60000",
+    "Height": "640",
+    "Width": "480"
+  },
+  "Actions": [
+    {
+      "Type": "FurmarkExecutor",
+      "Parameters": {
+        "Scenario": "ExecuteFurmarkBenchmark",
+        "PackageName": "Furmark",
+        "Time": "$.Parameters.Time"
+      }
+    }
+  ],
+  "Dependencies": [
+    {
+      "Type": "DependencyPackageInstallation",
+      "Parameters": {
+        "Scenario": "DownloadAMDDriverFromBlob",
+        "BlobContainer": "packages",
+        "BlobName": "AMDdependency.zip",
```

Comment:
Please rename the zip file in blob storage to something more descriptive of the driver being considered. For example, this driver is specific to Azure NVv4, so zip file can be something like AzureNvv4AMDGpuDrivers_22Q2.zip (depending on version)